vignettes/fieldworkanalysis_samara.Rmd

```diff
@@ -0,0 +1,95 @@
+---
+title: "Fieldwork Analysis"
+name: "Samara Khan"
+output: html_document
+---
+
+# Analysis objectives
+
+1. Import, recode, and subset data from bugsigdb.org    
+2. Create a table of studies
+3. Calculate the frequency of appearance of each taxa in independent signatures and identify the most frequently reported taxa
+4. Estimate the probability of the most frequently identified taxa occuring by chance
+
+#Making sure packages are installed
+
+```{r, eval=FALSE, messages=FALSE}
+install.packages(c("devtools", "tidyverse", "kableExtra"))
+devtools::install_github("waldronlab/bugSigSimple")
+devtools::install_github("waldronlab/BugSigDBStats")
+devtools::install_github("waldronlab/bugsigdbr")
+```
+
+#Load and subset data
+
+```{r, messages=FALSE}
+library(bugSigSimple)
+library(BugSigDBStats)
+library(bugsigdbr)
+dat <- bugsigdbr::importBugSigDB(cache= TRUE)
+dim(dat)
+names(dat)
+```
+
+#Subsetting only studies done on humans - 6 of 17 studies were excluded because they were either done on mice or not statistically reliable (Cregger et. al)
+
+```{r, messages=FALSE}
+library(tidyverse)
+library(dplyr)
+
+subset.dat <-dat[which(dat$PMID=="30778155" | dat$PMID=="32192080" | dat$PMID=="31087436"| dat$PMID=="26901400"| dat$PMID=="33839907"| dat$PMID=="32046455"| dat$PMID=="33925708"| dat$PMID=="32299442"| dat$PMID=="33313185"| dat$PMID=="34268384"| dat$PMID=="33660232"),]
```

Comment:
I just saw this! Make sure to tag @lwaldron or assign me to the review so I get a notification. This looks like a good start! Not a big deal but note you could do this simpler, such as:
```r
subset.dat <- filter(dat, PMID %in% c("30778155", "32192080"))
```